spherical_geometry/great_circle_arc.py

```diff
@@ -28,7 +28,7 @@
 
 
 def _inner1d_np(x, y):
-    return np.multiply(x, y).sum(axis=1)
+    return np.multiply(x, y).sum(axis=-1)
```

Comment:
So subtle. Does this need a regression test?

Reply:
Great idea! I'll do it in a week